client/content-plugins/ContainerRegistry.js

```diff
@@ -2,7 +2,8 @@ import ComponentRegistry from './ComponentRegistry';
 import containerList from 'shared/core-containers';
 import { getContainerName } from 'tce-core/utils';
 
-const ATTRS = ['type', 'version'];
-const options = ['container', containerList, ATTRS, getContainerName];
+const ATTRS = ['type', 'templateId', 'version'];
+const getCondition = templateId => it => it.templateId === templateId;
+const options = ['container', containerList, ATTRS, getContainerName, getCondition];
```

Comment:
A bit bloated now. We should probably transform it to object or group some params.